operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -34,4 +34,8 @@ if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq
   trace "Error: WebLogic Server state is FAILED_NOT_RESTARTABLE."
   exit $RETVAL
 fi
+if [ -f ${STATEFILE} ] && [ `grep -c "SHUTDOWN" ${STATEFILE}` -eq 1 ]; then
```

Comment:
Is this required for JRF?   Checking for shutdown would force a liveness probe failure after a normal shutdown, which leaves no approach for bringing down a server within a pod without causing the pod to immediately restart.
@mriccell @rjeberhard Please comment.

Reply:
no it is not required for jrf. i just did it in this same branch since the people wanting this are already using this branch. and making it restart the pod after you shut down the server is the desired behavior here.  it is only restarted if the domain CR says it should be running

Reply:
How can one shutdown WL and leave the pod running?

Reply:
@moreaut FYI

Reply:
why would you want to do that? what's the use case?

Reply:
I thought there was a use case to bring down WL, perform some local update using the container, bring up WL...

Reply:
I'm a bit confused - are you saying that if you go to the console and shutdown a server, then the operator will kill the pod and restart it?  If so, that doesn't sound intuitive.